org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/Messagebox.java

```diff
@@ -156,6 +170,17 @@ private void init()
 		btnNo.addEventListener(Events.ON_CLICK, this);
 		btnNo.setId("btnNo");
 		
+		String cancelTextLabel = Util.cleanAmp(Msg.getMsg(ctx, "Cancel"));
+		btnCancelText = ButtonFactory.createButton(cancelTextLabel, null, cancelTextLabel);
+		btnCancelText.addEventListener(Events.ON_CLICK, this);
+		btnCancelText.setId("btnCancelText");
+		
+		String deleteLabel = Util.cleanAmp(Msg.getMsg(ctx, "delete"));
+		btnDelete = ButtonFactory.createButton(deleteLabel, null, deleteLabel);
+		btnDelete.setStyle("color: white; background: red;");
```

Comment:
Setting the btnDelete and spanError in red hardcoded is not a good idea, it is better to do it in CSS.

But I think this comment would be obsolete, if we opt for the OK/Cancel button, then the Delete button is not necessary, and if we move to WrongValueException then the spanError is also unnecessary (making this comment useless).